config/base/params.env

```diff
@@ -24,3 +24,4 @@ MAX_CONCURRENT_RECONCILES=10
 DSPO_HEALTHCHECK_DATABASE_CONNECTIONTIMEOUT=15s
 DSPO_HEALTHCHECK_OBJECTSTORE_CONNECTIONTIMEOUT=15s
 DSPO_REQUEUE_TIME=20s
+DSPO_APISERVER_INCLUDE_OWNERREFERENCE=true
```

Comment:
this doesn't need to be optional, we always want to do this

Reply:
This needs to be optional so we can set it on tests. In production, it should never be set to `false`.
Is there a way to enforce it?

Reply:
oh right, you mentioned this earlier

not sure about enforcing it, but I'm fine with a simple comment explaining that this should always be enabled in production

Reply:
Done.